### PR TITLE
Update compliance criteria for missing data

### PR DESCRIPTION
**Pull request by philipwalton**
This PR clarifies how Core Web Vitals compliance handles cases where data is missing for a particular metric.

/cc: @rviscomi

### CODE REVIEW

**Comment by philipwalton** [comment]
FYI @egsweeny @addyosmani as this technically affects other tooling surfaces (though I don't think it's currently user-visible anywhere)

**Comment by kaycebasques** [comment]
Looks like we're waiting on reviews from @addyosmani @egsweeny @rviscomi

File: src/site/content/en/vitals/index.md
Comment (addyosmani) [inline src/site/content/en/vitals/index.md:76]: The verbiage we want to use here could be a little tighter (FID is the only metric where a lack of sufficient data allows you to pass as long as LCP and CLS are passing). We wouldn't consider a page as passing if FID and CLS were present, but say, LCP didn't have enough data.

**Comment by jpmedley** [comment]
@philipwalton We can't review this until you decided what to do about Addy's suggestion.

**Comment by kaycebasques** [comment]
@philipwalton waiting on you

**Comment by philipwalton** [comment]
I'm going to close this for now. We can revisit this change once we update various tooling related to this wording change.

**Review by cleytontiagosorinho — APPROVED** [review]
Só os aprovados

**Comment by rviscomi** [comment]
We should revisit this PR now that PSI takes optional FID into account.

**Comment by jpmedley** [comment]
@philipwalton @rviscomi 

This PR was made against master instead of mean which means I can't reopen it. Please submit a new PR. And please incorporate Addy's feedback.

**Pull request closed without merging**
